### ISSUE TITLE
Delete and redo button on previous text action missing on desktop browser

### ISSUE THREAD

**Issue by ghost** [ebolam/KoboldAI#306]
Hi there, Delete and redo button on previous text action missing on desktop browser even after refreshing the page. However, it is fine with my mobile browser. I attached a screenshot of the problem.
![Capture](https://user-images.githubusercontent.com/113113106/203464497-90576f86-1bf6-46ca-a44e-6b005941b14b.PNG)

**Comment by ebolam** [comment]
I'm not sure what I'm looking for to be honest. These look like generated text options. Those should have a pin icon with nothing else. If it's a redo (IE you hit back) then it should show with a circular arrow icon and a delete icon. 

What kind of options are these?

**Comment by ghost** [comment]
Hi there, for context, these are not currently generated text, these are previous story data and accumulates if I hit generate then undo for example. Let me try again later on a desktop to see if it repeats.

**Comment by ghost** [comment]
@ebolam I can confirm that it happens both on both my office and home desktop so, there really is something wrong, as you see this is historic action, not current action as I reproduce seven historic action with no delete option and redo.
![Screenshot 2022-11-26 091620](https://user-images.githubusercontent.com/113113106/204067088-baf41d7f-cc04-44d8-9904-d6d69fdbcda1.png)

**Comment by ghost** [comment]
@ebolam UPDATE: I can confirm this only happens on the UNICORN template preset, other presets are fine.

**Comment by ebolam** [comment]
I see the issue. They are still there, they're just white on white... Will have to figure out how to fix that

**Comment by ebolam** [comment]
Should be fixed now in 7141888163f298a1572bab2b7199d69a61df9bbe
